osmaxx-py/excerptconverter/gisexcerptconverter/gis_excerpt_converter.py

```diff
@@ -0,0 +1,259 @@
+        """
+        output_file = models.OutputFile.objects.create(
+            mime_type='application/zip',
+            extraction_order=extraction_order
+        )
+
+        if not os.path.exists(private_storage.location):
+            os.makedirs(private_storage.location)
+
+        file_name = str(output_file.public_identifier) + '.zip'
+        result_file_path = os.path.abspath(os.path.join(settings.RESULT_MEDIA_ROOT, result_file_name))
+        target_file_path = os.path.abspath(os.path.join(private_storage.location, file_name))
+
+        shutil.move(result_file_path, target_file_path)
+        output_file.file = private_storage.open(target_file_path)
+        output_file.save()
+
+        return os.path.isfile(target_file_path)
+
+    @staticmethod
+    @shared_task
+    def execute_task(extraction_order_id, supported_export_formats, execution_configuration):
+        wait_time = 0
+        # wait for the db to be updated!
+        extraction_order = None
+        while extraction_order is None:
+            try:
+                extraction_order = models.ExtractionOrder.objects.get(pk=extraction_order_id)
+            except models.ExtractionOrder.DoesNotExist:
+                time.sleep(5)
+                wait_time += 5
+                if wait_time > 30:
+                    raise
+
+        converter_helper = ConverterHelper(extraction_order)
+        extraction_order.state = models.ExtractionOrderState.WAITING
+        extraction_order.save()
+
+        with tempfile.TemporaryDirectory() as tmp_dir:
+            original_cwd = os.getcwd()
+
+            try:
+                print(tmp_dir)
+                shutil.copyfile(
+                    os.path.join(os.path.dirname(__file__), 'blackbox', 'docker-compose-conversion-blackbox.yml'),
+                    os.path.join(tmp_dir, 'docker-compose.yml')
+                )
+                os.chdir(tmp_dir)
+                # database needs time to be ready
+                subprocess.check_output("docker-compose run bootstrap sleep 10".split(' '))
+
+                converter_helper.inform_user(
+                    messages.INFO,
+                    _('The GIS extraction of the order "%s" is has been started.') % extraction_order.id,
+                    email=False
+                )
+
+                bounding_geometry = extraction_order.excerpt.bounding_geometry
+                if type(bounding_geometry) == models.BBoxBoundingGeometry:
+                    bbox_args = '%(excerpt_south_border)s %(excerpt_west_border)s %(excerpt_north_border)s ' \
+                        '%(excerpt_east_border)s' % {
+                            'excerpt_north_border': bounding_geometry.north,
+                            'excerpt_west_border': bounding_geometry.west,
+                            'excerpt_south_border': bounding_geometry.south,
+                            'excerpt_east_border': bounding_geometry.east
+                        }
```

Comment:
might be more readable as `bbox_args = ' '.join([bounding_geometry.north, bounding_geometry.west, bounding_geometry.south, bounding_geometry.east])`

Also, consider extracting a function for computing `bbox_args`.